components/Input/Input.tsx

```diff
@@ -0,0 +1,83 @@
+import { InputHTMLAttributes } from "react";
+
+interface InputProps
+  extends Pick<
+    InputHTMLAttributes<HTMLInputElement>,
+    | "id"
+    | "type"
+    | "value"
+    | "onChange"
+    | "name"
+    | "required"
+    | "className"
+    | "disabled"
+  > {
+  error?: boolean;
+  label: string;
+  helperText: string;
+}
+
+const Input = ({
+  id,
+  error = false,
+  label,
+  required,
+  helperText,
+  disabled,
+  ...props
+}: InputProps) => {
+  const spanStyles = `font-medium group-focus-within:font-bold ${
+    error ? "text-red font-bold" : ""
+  }`;
+  const inputStyles = `w-full py-[0.68rem] rounded-[3px] p-2 ring-1 mt-[0.44rem]
+  focus-visible:ring-[3px] outline-none ring-inset focus-visible:shadow-2md focus-visible:bg-blue-light focus:ring-orange 
+  disabled:ring-gray-dark disabled:bg-gray-light ${
+    error ? "ring-2 ring-red font-bold" : "ring-blue"
+  }`;
+  const svgStyles = `absolute right-5 top-11 ${error ? "block" : "hidden"}`;
+
+  return (
+    <div className="text-base font-inter space-y-1 group">
+      <label htmlFor={id} className="relative">
```

Comment:
Asi se ve el icono en Firefox (notese como el `label` toma forma como en dos partes)

![image](https://user-images.githubusercontent.com/78808163/220205386-1a81bf2e-7d73-489d-bb4c-79777a0d98fa.png)

---

Una solucion es agregar `block` al label 

```suggestion
      <label htmlFor={id} className="relative block">
```